rts/lra-examples/mixed-participant-with-coordinator/src/main/java/io/narayana/rts/lra/MixedResource.java

```diff
@@ -0,0 +1,92 @@
+package io.narayana.rts.lra;
```

Comment:
no copyright

Reply:
We should not have copyrights on quickstarts (the intent of a quickstart is for people to quickly get up and running by cutting and pasting and copyrights will hinder that approach).